java/src/main/bin/build.sh

```diff
@@ -147,6 +147,9 @@ install_brew_app "docker"
 docker-machine create --driver virtualbox default
 eval "$(docker-machine env default)"
 
+set -e
+set -u
+set -x
 run_docker "ubuntu:12.04" "$ubuntu_12"
 run_docker "32bit/ubuntu:14.04" "$ubuntu_14_32"
 run_docker "ubuntu:14.04" "$ubuntu_14"
```

Comment:
We can make the mac build happen faster by changing the last `make` to `make vw java`.